services/user.py

```diff
@@ -0,0 +1,56 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
```

Comment:
The import from db.models import User is redundant and potentially problematic. You're using Django's get_user_model() to fetch the user model, so importing User from your db.models isn't necessary and could cause confusion if User doesn't match the one returned by get_user_model()